### PR TITLE
Merge Release-1.1.0-small to Main

### PR DESCRIPTION
**Pull request by mreeve-snl**
Removes breaking changes in the release so dashboards and other QOL improvements can be released, and further testing/refactoring can be done on other release-1.1.0 code

## 🗣 Description ##

Removes the following files from the release-1.1.0, and resets them back to release 1.0 state. 
```
╰─$ git status
On branch release-1.1.0-small
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   Chapter 2 Files/SCCM Deployment/Uninstall_Sysmon64.ps1
        modified:   Chapter 3 Files/dashboard_update.sh
        modified:   Chapter 3 Files/deploy.sh
        modified:   Chapter 3 Files/docker-compose-stack.yml
        modified:   Chapter 3 Files/lme_update.sh
        modified:   docs/markdown/chapter3/chapter3.md
        modified:   docs/markdown/maintenance/upgrading.md
```

## 💭 Motivation and context ##

During testing, the current deployment process and setup creates difficulties and bottlenecks that were unexpected, and need to be refactored, improved, and updated. 

## 🧪 Testing ##

the dashboards and new script have been tested.

### CODE REVIEW

File: docs/markdown/chapter4.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ If you have never used Kibana before, Elasticsearch has provided a number of vid
 Kibana comes with many useful features. In particular, make note of the following:
 
 ### 4.3.1 Dashboards
-Found under both "Analytics" -> "Dashboard" and "Security" -> "Dashboard," dashboards are a great way to visualize LME data. LME comes with around many dashboards. Take some time to get familiar with the different dashboards already available. If interested in creating custom dashboards, see the link above for some starting points offered by Elasticsearch.
+Found under both "Analytics" -> "Dashboard," dashboards are a great way to visualize LME data. LME comes with several dashboards. Take some time to get familiar with the different dashboards already available. If interested in creating custom dashboards, see the link above for some starting points offered by Elasticsearch.
```
Comment (cbaxley) [inline docs/markdown/chapter4.md:106]: Remove word "both"

File: docs/markdown/reference/troubleshooting.md [hunk 2]
```diff
@@ -89,7 +89,53 @@ By default the `ForwardedEvents` maximum log size is around 20MB so events will
...
+
+You can try this:  [DISK-SPACE-20.04](https://askubuntu.com/questions/1269493/ubuntu-server-20-04-1-lts-not-all-disk-space-was-allocated-during-installation)
+```
+root@util:# vgdisplay
+root@util:# lvextend -l +100%FREE /dev/mapper/ubuntu--vg-ubuntu--lv
+root@util:~# resize2fs /dev/mapper/ubuntu--vg-ubuntu--lv
+```
+
+### Containers restarting/not running: 
+Usually if you have issues with containers restarting there is probably something wrong with your host or the container itself. Lik in the above sample, a wrong password could be prevent the Elastic Stack from operating properly. You can check the container logs like so:
```
Comment (cbaxley) [inline docs/markdown/reference/troubleshooting.md:106]: Change Lik to Like

**Review by cbaxley — CHANGES_REQUESTED** [review]

**Review by ddiabe — APPROVED** [review]

File: Chapter 3 Files/deploy.sh [hunk 3]
```diff
@@ -1033,21 +1043,17 @@ cd "$DIR" || exit
...
 elif [ "$1" == "renew" ]; then
   renew
+elif [ "$1" == "update" ]; then
+  update
 else
-  echo -e "\e[31m[!]\e[0m Invalid operation specified"
-  echo "Usage:    ./deploy.sh (install/uninstall/renew)"
-  echo "Example:  ./deploy.sh install"
-  exit 1
+  usage
```
Comment (mitchelbaker-cisa) [inline Chapter 3 Files/deploy.sh:1058]: Nice, keeping things DRY / reusable

File: docs/markdown/reference/troubleshooting.md [hunk 4]
```diff
@@ -89,7 +89,53 @@ By default the `ForwardedEvents` maximum log size is around 20MB so events will
...
+    volumes:
+      - type: volume
+        source: esdata
+        target: /usr/share/elasticsearch/data
+      - type: bind
+        source: /opt/lme/backups
+        target: /usr/share/elasticsearch/backups
+```
+
+To fix this you can change the permissions to what the conatiner expects:
```
Comment (mitchelbaker-cisa) [inline docs/markdown/reference/troubleshooting.md:130]: Container is misspelled

**Review by mitchelbaker-cisa — APPROVED** [review]

**Review by rgbrow1949 — APPROVED** [review]
Approved!

**Review by cbaxley — APPROVED** [review]

**Pull request merged**